quil-py/quil/instructions/__init__.pyi

```diff
@@ -360,73 +361,86 @@ class BinaryLogic:
     def operands(self, operands: BinaryOperands): ...
 
 class Calibration:
-    @staticmethod
     def __new__(
         cls,
         name: str,
-        parameters: List[Expression],
-        qubits: List[Qubit],
-        instructions: List[Instruction],
-        modifiers: List[GateModifier],
-    ) -> "Calibration": ...
+        parameters: Sequence[Expression],
+        qubits: Sequence[Qubit],
+        instructions: Sequence[Instruction],
+        modifiers: Sequence[GateModifier],
```

Comment:
Accepting `Sequence` instead of `List` makes these type hints more flexible. A `List` is a `Sequence`, and `pyo3` converts `Sequence`s to Rust `Vec`s, so this doesn't  betray the reality of the bindings.